edit-post/api/utils.js

```diff
@@ -0,0 +1,23 @@
+/* eslint no-console: [ 'error', { allow: [ 'error' ] } ] */
+
+/**
+ * Validates the plugin id.
+ *
+ * @param {string} id Plugin identifier.
+ * @return {boolean} Whether the pluginId is valid.
+ */
+export function validatePluginId( id ) {
```

Comment:
It looks like very similar code to what we have inside `registerBlockType:
https://github.com/WordPress/gutenberg/blob/master/blocks/api/registration.js#L97-L108

Should we move it to utils and make more general?